packages/components/psammead-consent-banner/src/index.jsx

```diff
@@ -80,35 +80,35 @@ const Options = styled.ul`
   list-style-type: none;
 
   & li + li {
-    padding-top: ${GEL_SPACING};
+    margin-top: 0.75rem;
+    float: ${ltrRtl('right', 'left')};
   }
 
-  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
-    width: 18%;
-    float: ${ltrRtl('right', 'left')};
+  @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
+    & li + li {
+      margin-top: 1rem;
+    }
   }
 `;
 
 const hoverFocusStyles = `
+  &:focus {
+    outline: none;
+    box-shadow: 0px 0px 1px 3px #68A1F8;
```

Comment:
Is this colour you are introducing in psammead-styles?

Reply:
We should always use `rem` units as these resizes based on user's browser configuration for accessibility etc

Reply:
Yeah that's the one, it's for the focus effect for buttons and links